test/SIM_test_varserv/models/varserv/src/VS_tests.cpp

```diff
@@ -33,112 +33,168 @@ int VSTest::strcmp_IgnoringWhiteSpace(const char* s1, const char* s2) {
     }
 }
 
-
+// Not really sure what this is supposed to be testing, should probably just remove
```

Comment:
Yes, you can remove anything in the VS tests that you don't think is meaningful.

Reply:
Removed this and some other commented out code in VS test